packages/expo-face-detector/src/FaceDetector.ts

```diff
@@ -52,6 +52,15 @@ export type DetectionOptions = {
   runClassifications?: FaceDetectorClassifications;
 };
 
+/**
+ * Returns whether the Face Detection API is enabled on the current device. This does not check the app permissions.
+ *
+ * @returns Async `boolean`, indicating whether the Face Detection API is available on the current device. Currently this resolves `true` on iOS and Android only.
+ */
+export async function isAvailableAsync(): Promise<boolean> {
+  return !!ExpoFaceDetector.detectFaces;
```

Comment:
Right now `ExpoFaceDetector` can be a `null` (if the native code isn't properly linked, note there's no fallback to `{}` in `ExpoFaceDetector.ts`)

https://github.com/expo/expo/blob/8d441c7e9b081cfdd6a45edc165251d7273225a0/packages/expo-face-detector/src/ExpoFaceDetector.ts#L1-L3

Maybe we should do something like `ExpoFaceDetector && 'detectFaces' in ExpoFaceDetector`?

Reply:
Good point, how do you feel about the `|| {}` fallback? I think it would simplify the lookup, and matches web better.